src/algebra/group_power/order.lean

```diff
@@ -180,6 +180,9 @@ monotone_nat_of_le_succ $ λ n,
 theorem pow_le_pow (ha : 1 ≤ a) (h : n ≤ m) : a ^ n ≤ a ^ m :=
 pow_mono ha h
 
+theorem le_pow (ha : 1 ≤ a) (h : 1 ≤ m) : a ≤ a ^ m :=
```

Comment:
Perhaps:
```suggestion
theorem le_self_pow (ha : 1 ≤ a) (h : 1 ≤ m) : a ≤ a ^ m :=
```

Is it worth adding the similar statement for `a ≤ 1`? That would likely inform what a good choice of name is.

Reply:
Also, I get the feeling the preferred spelling was `0 < m` or `m ≠ 0` instead of `1 ≤ m`

Reply:
@eric-wieser We considered adding the similar statement for `a ≤ 1` but we believe a few more lemmas would be required to support this. We would need an analogous (pow_anti) theorem to pow_mono. Is this what you had in mind?